src/sentry/rules/conditions/level.py

```diff
@@ -58,7 +58,7 @@ def passes(self, event, state, **kwargs):
             return False
 
         desired_level = int(desired_level)
-        level = int(event.level)
+        level = int(event.group.level)
```

Comment:
Semi-unrelated, but this is where the original bug came from. This method is just deprecated, so switching it to use the un-deprecated way.